src/ClientData/include/ClientData/CallstackData.h

```diff
@@ -96,12 +96,12 @@ class CallstackData {
   // E.g., one might want to nest ForEachCallstackEvent and ForEachFrameInCallstack.
   mutable std::recursive_mutex mutex_;
   absl::flat_hash_map<uint64_t, std::shared_ptr<orbit_client_protos::CallstackInfo>>
-      unique_callstacks_ GUARDED_BY(mutex_);
```

Comment:
I guess it doesn't work with std::mutexes...

Reply:
At least we can't ensure to compile against a libstdc++ that has annotated header files. And since Abseil has no recursive mutex it's best to remove it. (Also documented in the commit message)